FluentTc.Tests/AcceptanceTests.cs

```diff
@@ -108,8 +108,46 @@ public void GetBuildFullResponse_Id_Build()
 
             // Assert
             build.Id.Should().Be(123);
-        }        
-        
+        }
+
+        [Test]
+        public void GetBuildFullResponse_TestOccurrences_Build()
+        {
+            // Arrange
+            var teamCityCaller = CreateTeamCityCaller();
+            A.CallTo(() => teamCityCaller.Get<BuildModel>("/app/rest/builds/id:123")).Returns(
+                new BuildModel
+                {
+                    Id = 123,
+                    Status = "SUCCESS",
+                    TestOccurrences = new TestOccurrences
+                    {
+                        Count = 13,
+                        Href = "/app/rest/testOccurrences?locator=build:(id:123)",
+                        Passed = 1,
+                        NewFailed = 2,
+                        Failed = 3,
+                        Muted = 4,
+                        Ignored = 5,
+                    }
+                });
+
+            var connectedTc = new RemoteTc().Connect(_ => _.AsGuest(), teamCityCaller);
+
+            // Act
+            var build = connectedTc.GetBuild(123);
+
+            // Assert
+            build.TestOccurrences.Should().NotBe(null);
```

Comment:
There is a dedicated method on FluentAssertions to validate object for not being null

```C#
obj.Should().NotBeNull();
```